src/kraft.erl

```diff
@@ -45,8 +74,58 @@ start(Opts, Routes) ->
     }),
     Ref.
 
-stop(Ref) -> kraft_instance:stop(Ref).
+%%------------------------------------------------------------------------------
```

Comment:
Can you remove these delimiters? I'd like them only between "sections" (e.g. types, macros, exported functions, internal functions etc.)